DigitalLearningSolutions.Data/Helpers/ConfigHelper.cs

```diff
@@ -14,6 +14,7 @@ public static class ConfigHelper
         private const string LearningHubSsoToleranceKey = "ToleranceInSeconds";
         private const string LearningHubSsoIterationsKey = "HashIterations";
         private const string LearningHubSsoByteLengthKey = "ByteLength";
+        private const string LearningHubSsoSecretKey = "SecretKey";

```

Comment:
Can this also be placed into the `appSettings.json` file with a "fake" value set?